libs/terraform_utils/plan_summary.go

```diff
@@ -132,3 +138,17 @@ func SimilarityCheck(footprints []TerraformPlanFootprint) (bool, error) {
 	return allSimilar, nil
 
 }
+
+func GetTfSummarizePlan(planJson string) (string, error) {
+	plan := tfjson.Plan{}
```

Comment:
wow TIL that we can rely on the types in the library rather than bake our own types as in this file!

Reply:
It's pretty neat :) I think that OpenTofu still uses `github.com/hashicorp/terraform-json`, so it looks like its not forked yet.